### DISCUSSION TITLE
Function transfer

### DISCUSSION THREAD

**Discussion by sash22** [rxhanson/Multitouch-Community#217] [Ideas]
Hi, is it possible to add the window placement functionality from Rectangle Pro? I bought an external monitor, and now I have to use App Layouts from Rectangle Pro in addition to this program, and this is the only function I need from Rectangle.

**Comment by rxhanson** [comment]
Thanks for the feedback! I may consider moving this over if I end up merging nearly all of Rectangle Pro into Multitouch, since it's a pretty large piece of functionality. I don't have immediate plans to do this, but I haven't ruled it out.
